scripts/obtain_customer_cc.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+from genologics.entities import *
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+
+def main(lims, args):
+    pro=Process(lims, id=args.pid)
+    for inp in pro.all_inputs():
+        updated=False
+        if 'Customer Conc' in inp.samples[0].udf:
+            inp.udf['Concentration']=inp.samples[0].udf['Customer Conc']
```

Comment:
This PR looks good enough to me, in that you just copy some values around. I have no idea of the structure of the process, and why you would do this. So forgive my ignorance:
`samples[0]` -- Why not `for sample in samples` ?